test/cli.js

```diff
@@ -0,0 +1,107 @@
+var chai = require('chai');
+var child = require('child_process');
+var util = require('./util');
+var exec = child.exec;
+var expect = chai.expect;
+
+describe('cli', function() {
+  var input = util.read('fixtures/cli/input');
+  var output = util.read('fixtures/cli/input.out');
+
+  afterEach(function() {
+    util.remove('fixtures/cli/output');
+  });
+
+  it('should read from a file and write to a file', function(done) {
+    exec('node bin/suitcss -c test/config/noautoprefixer.config.js test/fixtures/cli/input.css test/fixtures/cli/output.css', function(err) {
```

Comment:
I did have the same thought about some other areas of the code.  [The coverage](https://kangax.github.io/compat-table/es6/#node4) in Node 4 is patchy, but I think we can use arrows, object literal extensions and template literals.

It requires ESLint to be refactored too.